lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -15,23 +15,17 @@ class PointerBinding {
   static PointerBinding get instance => _instance;
   static PointerBinding _instance;
 
-  // Set of pointerIds that are added before routing hover and mouse wheel
```

Comment:
we no longer need to guess whether the framework will synthesize an add or not, because it won't.